### ISSUE TITLE
Nodemon cannot find new routes after added them.

### ISSUE THREAD

**Issue by kelvindemirandabarros** [remy/nodemon#1740]
- `nodemon -v`: 2.0.4
- `node -v`: 14.5.0
- Operating system/terminal environment: Windows / Git Bash
Edition: Windows 10 Pro
Version: 1909
Installed on: 02/29/2020
OS Build: 18363.900

- Using Docker? What image: Yes / Postgres
- Command you ran: `nodemon src/server.mjs`

### Expected behaviour

I have these three simple files in the 'src' paste:

- ROUTES.MJS
```
import Router  from 'express';
const routes = new Router();
routes.get( '/', ( req, res ) => {
    return res.json({ message: 'Tudo ok! / Everything ok!' });
});
routes.get( '/2', ( req, res ) => {
    return res.json({ message: 'Tudo ok! / Everything ok!' });
});
export default routes;
```

- APP.MJS
```
import express from 'express';
import routes from './routes.mjs';
class App {
    constructor () {
        this.server = express();
        this.routes();
    }
    routes () {
        this.server.use( routes );
    }
}
export default new App().server;
```

- SERVER.MJS
```
import server from './app.mjs';
const port = 5000;
server.listen( port, () => {
    console.log( `Servidor ligado na porta: / Server running on port:  ${ port }.` );
});
```

When I have only the first route ('/') and start the server, I can access it, and when I added the second route ('/2') and save the file, the server should be restarted automatically and I could access the second route ('/2').

### Actual behaviour

The server restarts, but I still can not access the second route ('/2'). Only when I restart the server manually.

### Steps to reproduce

Listed above.

Basically, adding a new route to the project will not have effect until shutting down the server and running it again, manually.

**Comment by AlanVncs** [comment]
Is the node able to run typescript code? Shouldn't you compile your code before running it again?
If your "npm start" script have a compile command, it will execute at the first time, but when nodemon restart, the compile command will not run again and your compiled files will not change.
If I'm right, I can show you a nice way to handle it.

**Comment by kelvindemirandabarros** [comment]
How can I know if node can run Ts? And I do not know if the code compiles everything to run it. I am running the server with Yarn. Can you talk about it? Is there any other info that you need to get to the problem?

**Comment by AlanVncs** [comment]
What command do you use to run the server? `yarn start`?
Can you post here the section "scripts" inside you package.json?

**Comment by kelvindemirandabarros** [comment]
```
"scripts": {
    "dev": "nodemon src/server.mjs"
}

- yarn run dev
```

The basic.
